cl_dll/hud_redraw.cpp

```diff
@@ -201,10 +201,17 @@ int CHud :: Redraw( float flTime, int intermission )
 
 			if (m_Speedometer.m_iFlags & HUD_ACTIVE)
 				m_Speedometer.Draw(flTime);
+			
+			if (m_Jumpspeed.m_iFlags & HUD_ACTIVE)
+				m_Jumpspeed.Draw(flTime);
```

Comment:
what's this doing here

Reply:
oh, I see, can you move it into a separate commit?

Reply:
Sure